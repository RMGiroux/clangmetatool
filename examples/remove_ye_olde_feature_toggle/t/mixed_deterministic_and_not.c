#include <ye_olde_feature_toggle.h>
int test_function() {
  if (ye_olde_feature_toggle_is_enabled(15) ||
      ye_olde_feature_toggle_is_enabled(3)) {
    return 1;
  } else {
    return 0;
  }
}
