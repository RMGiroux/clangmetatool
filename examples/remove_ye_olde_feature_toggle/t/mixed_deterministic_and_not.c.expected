#include <ye_olde_feature_toggle.h>
int test_function() {
  if (ye_olde_feature_toggle_is_enabled(15) ||
      1 /* removed ye_olde_feature_toggle 3 */) {
    return 1;
  } else {
    return 0;
  }
}
